lib/plugins/careportal.js

```diff
@@ -10,6 +8,91 @@ function init() {
     , pluginType: 'drawer'
   };
 
+  careportal.getEventTypes = function getEventTypes (sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['sbx' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1503885462/issues/source?bid=2452379&fileBranchId=2989096#l11)